specs/packagemanager/PackageManagement.md

```diff
@@ -635,6 +644,66 @@ void CheckAndFixPackageVolume(string packageStorePath)
 }
 ```
 
+## 4.9. IsPackageProvisioned()
+
+Fabrikam app installing Contoso's Muffin and Waffle packages for all users if necessary, and with explicit user confirmation before the installation.
+
+```c#
+void Install()
+{
+    var packageSet = new PackageSet() {
+        Items = { new PackageSetItem() { PackageFamilyName = "contoso.muffin_1234567890abc",
+                                         MinVersion = ToVersion(1, 2, 3, 4),
+                                         PackageUri = new Uri("c:\\contoso\\muffin-1.2.3.4.msix") },
+                { new PackageSetItem() { PackageFamilyName = "contoso.waffle_1234567890abc",
+                                         MinVersion = ToVersion(2, 4, 6, 8),
+                                         PackageUri = new Uri("https://contoso.com/waffle-2.4.6.8.msix") } };
+
+    var packageDeploymentManager = PackageDeploymentManager.GetDefault();
+    if (!packageDeploymentManager.IsPackageSetProvisioned(packageSet))
+    {
+        bool ok = PromptUserForConfirmation();
+        if (!ok)
+        {
+            return;
+        }
+    }
```

Comment:
>} [](http://example.com/codeflow?start=4&length=1)

Do we need to do anything if packageDeploymentManager.IsPackageSetProvisioned(packageSet) returns true I thought that means that there was no need to provision the package.

Reply:
Doh! Copy/paste error. Yes, we should provision

RECOMMEND: Fix the sample

Reply:
still missing?